app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,14 @@
+class UserMailer < ApplicationMailer
+
+  def registration_success(user)
+    @user = user
+    mail(to: @user.email, subject: 'Welcome to MemryBook')
```

Comment:
No need to assign the parameter to a instance variable, you can straightaway use the parameter in the method. Please define message in en.yml